### PR TITLE
Point readers at the repo that uses the packages produced by this one.

### LINKED ISSUE #66: Document how to deploy a federated architecture
I had the impression from the talk at the UGM there were some scripts to deploy a federated set of zones which could be used for testing e.g. upgrades, but I don't see them in this repository, have I missed them?

Also a section on how to spin them up would be helpful - there is an example of how to run a single docker container, but not the wider infrastructure.

### PR DESCRIPTION
**Pull request by kript**
Fixes #66

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This repository contains tools for the running and troubleshooting of a Docker-containerized iRODS server.
 
+It is used as part of the [irods_testing_environment](https://github.com/irods/irods_testing_environment), which can use the packages produced by this repo, if you wanted to build them from scratch.
```
Comment (trel) [inline README.md:5]: Consider, instead...

"The [irods_testing_environment](https://github.com/irods/irods_testing_environment) can be used to test the packages produced by this repository."

**Comment by trel** [comment]
Perfect - can you squash to a single commit and we'll merge it.    What does the 'Fixes' in your commit description mean?

**Comment by kript** [comment]
> Perfect - can you squash to a single commit and we'll merge it. What does the 'Fixes' in your commit description mean?

I _think_ I've done that, but I am still an n00b when it comes to `rebase`, so apologies if I've screwed something up! 

The 'fixes' line was supposed to indicate that it addresses the linked issue (and might close it when done? Less confident of that!).

**Comment by trel** [comment]
You got it to one commit - but both commit messages were combined - you can now run `git commit --amend` and it will take you to an editor to update the one combined commit message.  Then you would force push again to your branch and the PR will update again.

There is no linked issue - just a pointer to the other repository.   To link something there, add a # and the issue number... you can use the preview tab to see if you get it right.

**Comment by trel** [comment]
Ah, I see as well - in that case... please start the commit message itself with that information - so just "[66] ", but with the pound sign so it links.

Then, force push once more... and it looks like you got it in one!

**Comment by korydraughn** [comment]
What @trel is saying is ... run `git commit --ammend` and make the commit message use the following structure:
```
[#66] Your summary of the change goes here ...
```
If you wanted to link to an issue in another repo, you'd tweak the message to use the following prefix:
```
[irods/other_repo#66] Your summary of the change goes here ...
```
Then you just do a force-push to update the PR.

**Comment by kript** [comment]
> What @trel is saying is ... run `git commit --amend` and make the commit message use the following structure:
> 
> ```
> [#66] Your summary of the change goes here ...
> ```
> 
> If you wanted to link to an issue in another repo, you'd tweak the message to use the following prefix:
> 
> ```
> [irods/other_repo#issue_number] Your summary of the change goes here ...
> ```
> 
> Then you just do a force-push to update the PR.

I _think_ I have done that, at leas as per the commit history?

**Comment by trel** [comment]
Yes, you've got one commit, correctly, and it's linking, correctly... but the commit message isn't helpful :P

So, we're looking for...

```
[#66] John describes the commit here ...
```

**Comment by alanking** [comment]
@kript Yes, but, the commit history is erased (on purpose) in the rebase/squash paradigm, so we aren't seeing any of the commit message(s) you may or may not have written before. Just mentioning for extra info in case it's helpful :)

**Pull request merged**